pom.xml

```diff
@@ -3,11 +3,9 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-
```

Comment:
please don`t change pom.xml

Reply:
Sorry, but I probably will not be able to return as it was before. But it doesn't seem to affect the program in any way.

Reply:
I don`t see any problems to rollback this changes. Yes it does not affect program but it doest not mean that you can thange some files just becuse you want so. This changes will be tracked by git blame and  developer who will investigate changes in pom will be confused what happend will search what was the reason of change in scope of this task and in the result after some time maybe 20 mins maybe an hour he will find out chat you change file just because. For you it takes a minute to rollback changes, so please do that